test/inductor/test_torchinductor_opinfo.py

```diff
@@ -392,6 +392,41 @@ def wrapper_set_seed(op, *args, **kwargs):
     torch.manual_seed(42)
     return op(*args, **kwargs)
 
+# OpInfo that call wrapper_set_seed.
+# Ref: https://github.com/pytorch/pytorch/issues/107187
+# Since then, manual_seed calls graph-breaks. Thus, in that case, we need to allow
+# Python code.
+inductor_nopython_error = {
+    "item",
+    "cauchy",
+    "exponential",
+    "geometric",
+    "log_normal",
+    "normal",
+    "normal.in_place",
+    "normal.number_mean",
+    "uniform",
+    "nn.functional.fractional_max_pool2d",
+    "nn.functional.fractional_max_pool3d",
+    "nn.functional.rrelu",
+    "nn.functional.scaled_dot_product_attention",
+    "svd_lowrank",
+    "pca_lowrank",
+    "randn",
+    "randn_like",
+    "rand_like",
+    "randint",
+    "randint_like",
+    "empty_strided",
+    "multinomial",
+    "bernoully",
+    "nn.functional.dropout",
+    "nn.functional.dropout2d",
+    "nn.functional.dropout3d",
+    "nn.functional.alpha_dropout",
+    "nn.functional.feature_alpha_dropout",
+    "nn.functional.multi_head_attention_forward",
+}
```

Comment:
There are many tests failing because they called `torch.manual_seed` in their `OpInfo`. Since we are graph-breaking on `manual_seed` calls, we need to allow Python code for them.

Reply:
Can we instead monkey-patch `manual_seed` to be a no-op in the tests, and then set manual_seed ourselves in the test in torchdynamo ?

Reply:
Hmm. Apparently, it worked without `fallback_random`, but simply making `wrapper_set_seed` a noop. Am I missing anything?

Reply:
The opfinos for all the rng functions expect failures right now: https://github.com/pytorch/pytorch/blob/main/test/inductor/test_torchinductor_opinfo.py#L281

I think why it worked

Reply:
But, take a look at this part of `check_model` function. We already set `torch.manual_seed(0)` before calling both the plain model and the optimized one. So, maybe this is not the problem!?

https://github.com/pytorch/pytorch/blob/c68d0a7042e850cebc4cbe7f717fc11aedf6b9d7/test/inductor/test_torchinductor.py#L289-L312

For `randint` and some others (guess), I think they have different pseudo-generating algorithms:
- CPU eager: https://github.com/pytorch/pytorch/blob/main/aten/src/ATen/CPUGeneratorImpl.h
- CPU Inductor: https://github.com/pytorch/pytorch/blob/main/aten/src/ATen/core/PhiloxRNGEngine.h

Plus, as far as I've seen, `randint` doesn't seem to get its seed from the `default_generator` (which is the one affected  by `manual_seed`).